src/core/config/type-cast/type-casters/combined.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @prettier
+ */
+const combinedTypeCaster = (firstTypeCaster, secondTypeCaster) => (value) =>
```

Comment:
Check the https://lodash.com/docs/4.17.15#flow or https://lodash.com/docs/4.17.15#flowRight from `lodash`.